packages/pn-pa-webapp/src/pages/__test__/NewNotification.page.test.tsx

```diff
@@ -33,38 +32,47 @@ jest.mock(
       )
 );
 
-jest.mock(
-  '../components/NewNotification/Recipient',
-  () =>
-    ({ onConfirm }: { onConfirm: () => void }) =>
-      (
-        <div data-testid="stepContent">
-          Recipient <button onClick={onConfirm}>Click Me!</button>
-        </div>
-      )
-);
+jest.mock('../components/NewNotification/Recipient', () => {
```

Comment:
Please add some comment explaining the motivation for this change, thanks!

Reply:
Done